forest-node/ansible/roles/docker/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
There are some tasks that may require docker restart in the future (maybe) so we make docker flexible for restart at any time. Generally, handlers are used to maybe start, reload, restart, and stop running services. 

If the playbook involves changing configuration files, there is a high chance that we might need to restart a service so that the changes take effect. We can add maybe a trigger word `run`, `notify`, or something in the tasks that need the service restart before making the tasks successful.  

So once again, this very handler is about best practice.